src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -533,6 +533,7 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = ({
           setValue={setValue}
           error={errors.bannerImageFile || errors.bannerImageUrl}
           isInputHidden={isBtnOnly}
+          btnLabel="Upload Highlight image"
```

Comment:
```suggestion
          text="Upload Highlight image"
```

or a buttonText or anything but `btnSomething` (to avoid hungarian notation based on the type of the variable).

Reply:
fixed 65d922a